go/store/prolly/tree/node.go

```diff
@@ -223,14 +223,14 @@ func OutputProllyNode(ctx context.Context, w io.Writer, node Node, ns NodeStore,
 				w.Write([]byte(", "))
 			}
 
-			isAddr := val.IsAddrEncoding(kd.Types[i].Enc)
```

Comment:
This is kind of alarming, good catch.

Reply:
I think this code only gets hit in tests, and only when tests fail. So we weren't seeing it.